packages/yoastseo/src/languageProcessing/helpers/word/markWordsInSentences.js

```diff
@@ -76,14 +77,18 @@ const getMarkedAnchors = function( sentence, wordsRegex ) {
  * @returns {string} The sentence with marks.
  */
 export const collectMarkingsInSentence = function( sentence, wordsFoundInSentence, matchWordCustomHelper ) {
-	wordsFoundInSentence = wordsFoundInSentence.map( word => escapeRegExp( word ) );
+	wordsFoundInSentence = wordsFoundInSentence.map( word => {
+		word = escapeRegExp( word );
+		word = normalizeSingle( word );
```

Comment:
Early CR: I would consider switching around these operations, I would not change an escaped word anymore 😄